src/Microsoft.DotNet.Interactive.NamedPipeConnector/Microsoft.DotNet.Interactive.NamedPipeConnector.csproj

```
@@ -0,0 +1,30 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net7.0</TargetFramework>
```

This probably should be net7.0-windows 

```suggestion
    <TargetFramework>net7.0-windows</TargetFramework>
```